andela/getmyshop
# #122422621 Shop Creation

#### What does this PR do?

Allows shop creation by shop owners
#### Description of Task to be completed?

Users should be able to register as shop owners and create shops when their account is activated.
#### How should this be manually tested?

As a shop owner
When I visit the homepage
And click on register
Then I should be redirected to the registration page
When I fill the registration data
Then I should be registered and redirected to the dashboard
#### What are the relevant pivotal tracker stories?
#122422621
